src/classes/CON_PreferredPhone_Test.cls-meta.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="urn:metadata.tooling.soap.sforce.com" fqn="CON_PreferredPhone_Tesst">
```

Comment:
Misspelled "Test" here, though I don't know how much this matters.

However, I would suggest we change the file time of these classes to "_TEST", all capitals. The rest of the test classes follow this convention and I'm not sure if CumulusCI is case sensitive or not when it queries for all test classes to run during the build.